examples/meson.build

```diff
@@ -0,0 +1,2 @@
+subdir('cpp')
+subdir('fortran')
```

Comment:
Note: we should also add the "notebooks" and "python" directories as well, after we add the meson.build files to these.